### PR TITLE
Add network case of qos

### PR DESCRIPTION
**Pull request by chloerh**
- VIRT-296327 - [Qos] Check the actual network throughput per bandwidth settings in network/interface

### CODE REVIEW

File: libvirt/tests/src/virtual_network/qos/check_actual_network_throughput.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        if not utils_net.check_class_rules(*tc_args):
+            test.fail('Class rule check failed. Please check log.')
+        LOG.debug('Class rule check: PASS')
+
+        if not utils_net.check_filter_rules(tc_args[0], filter_bw):
+            test.fail('Filter rule check failed. Please check log.')
+        LOG.debug('Filter rule check: PASS')
+
+        if not utils_package.package_install('netperf', session=vm_sess):
+            test.error('Failed to install netperf on VM.')
```
Comment (Yingshun) [inline libvirt/tests/src/virtual_network/qos/check_actual_network_throughput.py:92]: If it fails, can we compile via virttest.utils_netperf.NetperfPackage().install()?

File: provider/virtual_network/network_base.py [hunk 2]
```diff
@@ -336,3 +337,29 @@ def check_iface_attrs(iface, key, expect_val):
...
+    :param cli_runner: runner of netperf client
+    :param ip_addr: ip address
+    :param bw: bandwidth setting
+    :param th_type: inbound or outbound
+    """
+
+    serv_runner('netserver')
+    netperf_out = cli_runner(f'netperf -H {ip_addr}')
+    LOG.debug(netperf_out)
+    serv_runner('pkill netserver')
```
Comment (Yingshun) [inline provider/virtual_network/network_base.py:356]: reuse utils_netperf.NetperfServer() and utils_netperf.NetperfClient()?

File: libvirt/tests/src/virtual_network/qos/check_actual_network_throughput.py (hunk 1 above)
Comment (chloerh) [reply libvirt/tests/src/virtual_network/qos/check_actual_network_throughput.py:92]: Seems the `install()` function needs to create a session itself, not supporting passing a session as param for now, maybe after the function being updated?

File: provider/virtual_network/network_base.py (hunk 2 above)
Comment (chloerh) [reply provider/virtual_network/network_base.py:356]: I've checked this too, still needs to support vm_session.

**Review by yalzhang — COMMENTED** [review]
local run pass

**Review by yalzhang — APPROVED** [review]

**Review by Yingshun — APPROVED** [review]

**Pull request merged**